config.toml

```diff
@@ -445,6 +445,10 @@ logo = "teamleader.png"
 url = "https://zapier.com"
 logo = "zapier.png"
 
+[[params.users]]
+url = "https://www.bolddesk.com"
+logo = "bolddesk.svg"
```

Comment:
Sorry for the nit, but can you alphabetically sort it please?

Reply:
i have reordered it alphabetically, can you review now?